docs/frontend/in-depth/autosave.md

```diff
@@ -0,0 +1,154 @@
+# Autosave
+
+Autosave is the way of saving objects in system. Main idea about that is to prevent user from leaving any page without saved data and to make sure made changes are conscious. The idea is build at top of:
```

Comment:
to make sure that changes he/she made are intentional. The idea is to build on top of: